geode-core/src/distributedTest/java/org/apache/geode/distributed/DistributedLockServiceDUnitTest.java

```diff
@@ -1639,8 +1707,10 @@ public void run() {
    * old dlock throws StuckLockException. VM-C should now succeed in acquiring the lock.
    */
   @Test
-  public void testBug32461() throws Exception {
-    logger.debug("[testBug32461] prepping");
+  public void testBug32461() {
+    if (logger.isDebugEnabled()) {
```

Comment:
My understanding is that this test is done in product code for performance - `logger.debug()` has a higher overhead then testing `logger.isDebugEnabled()`. So, using this conditional in tests is not usually necessary.

Reply:
Thanks for the info! I just assumed that it was needed based on Darrel's comment but good to know that I don't have to do that in the future.